common/pb/serverinfo_chat_message.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
+/*
+ * Chat communication of a user to a target.
+ * Targets can be users or rooms.
+ * These communications are also stored in the DB log table.
+ */
+message ServerInfo_ChatMessage {
+    optional string time = 1; // time chat was sent
+    optional string sender_id = 2; // id of sender
+    optional string sender_name = 3; // name of sender
+    optional string sender_ip = 4; // ip of sender
```

Comment:
Gotcha, that makes sense.

I don't have a problem with using it this way, but it is always free to add new optional fields later.